src/brain.py

```diff
@@ -47,7 +54,7 @@ def get_valid_moves(self) -> List[str]:
 
     def get_nearest_food(self) -> Optional[Coordinate]:
         """Get the food item which has coordinates closest to this snake's head."""
-        closest_food = (Coordinate((0,0)), 9999.0)
+        closest_food = (Coordinate((0,0), None), 9999.0)
```

Comment:
I think this will need to be reverted based on the changes to `Coordinate` in the other PR

- [ ] Remove `None`